test_zonos.py

```
@@ -0,0 +1,572 @@
+            activities=[    
+                torch.profiler.ProfilerActivity.CPU,
+                torch.profiler.ProfilerActivity.CUDA,
+            ],
+            on_trace_ready=torch.profiler.tensorboard_trace_handler("./profile_logs"),
+            record_shapes=True,
+            profile_memory=True,
+            with_stack=True,
+            with_modules=True,
+        ) as prof:
+            def mark_step(*args) -> bool:
+                prof.step
```

Ruff (0.12.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)